.travis.yml

```diff
@@ -37,6 +37,7 @@ install:
   - psql -U postgres -c 'create database "jarbas";'
   - yarn install
   - python -m pip install -r requirements.txt coveralls
+  - python -m pip install -r requirements-test.txt coveralls
```

Comment:
We're trying to install `coveralls` twice. I'd change the strategy here: 

* Rename `requirements-test.txt` to `requirements-dev.txt`
* Include `-r requirements.txt` as the first line of in the new `requirements.txt`
* Move `django-test-without-migrations` from `requirements.txt` to `requirements-dev.txt`
* Use _only_ `requirements-dev.txt` in `.travis.yml`